### PR TITLE
add lookup cache for daemonset

### PR DESCRIPTION
**Pull request by mqliang**

### CODE REVIEW

**Comment by mqliang** [comment]
@wojtek-t

**Comment by bprashanth** [comment]
@mikedanese

File: pkg/apis/componentconfig/types.go [hunk 1]
```diff
@@ -431,6 +431,9 @@ type KubeControllerManagerConfiguration struct {
 	// LookupCacheSizeForRS is the size of lookup cache for replicatsets.
 	// Larger number = more responsive replica management, but more MEM load.
 	LookupCacheSizeForRS int `json:"lookupCacheSizeForRS"`
+	// LookupCacheSizeForDaemonSet is the size of lookup cache for daemonsets.
```
Comment (mikedanese) [inline pkg/apis/componentconfig/types.go]: godoc in this file should match field name, so lookupCacheSizeForDaemonSet

**Comment by mqliang** [comment]
@mikedanese Updated. PTAL.

File: pkg/apis/componentconfig/types.go (hunk 1 above)
Comment (mikedanese) [inline pkg/apis/componentconfig/types.go]: Thanks!

**Comment by wojtek-t** [comment]
LGTM

**Comment by mqliang** [comment]
@wojtek-t Rebased.

**Comment by wojtek-t** [comment]
@mqliang - can you please rebase?

**Comment by mqliang** [comment]
@wojtek-t Rebased. PTAL

**Comment by wojtek-t** [comment]
LGTM - thanks!

**Comment by ncdc** [comment]
We're tracking this for OpenShift - any objections to adding the v1.2 milestone to this PR? @thockin @bgrant0607 @mikedanese @davidopp

**Comment by davidopp** [comment]
We don't have milestones for PRs. Since it has been LGTMd, it should merge by the time we cut the branch on Friday.

**Comment by ncdc** [comment]
Discussed with @davidopp on slack; it's ok to add the milestone.

**Comment by bgrant0607** [comment]
We do have milestone for PRs. The submit queue is going to prioritize 1.2 above other PRs.

**Comment by bgrant0607** [comment]
Also, without a priority, there's no way this will merge by Friday morning.

**Comment by bgrant0607** [comment]
Does this provide measurable benefit?

**Comment by wojtek-t** [comment]
@k8s-bot e2e test this please github issue: #IGNORE (tests are pending)

**Pull request merged**